includes/modules/pages/shopping_cart/header_php.php

```diff
@@ -132,7 +132,16 @@
   $ppt = $ppe * $products[$i]['quantity'];
   $productsPriceEach = $currencies->format($ppe) . ($products[$i]['onetime_charges'] != 0 ? '<br />' . $currencies->display_price($products[$i]['onetime_charges'], zen_get_tax_rate($products[$i]['tax_class_id']), 1) : '');
   $productsPriceTotal = $currencies->format($ppt) . ($products[$i]['onetime_charges'] != 0 ? '<br />' . $currencies->display_price($products[$i]['onetime_charges'], zen_get_tax_rate($products[$i]['tax_class_id']), 1) : '');
-  $buttonUpdate = ((SHOW_SHOPPING_CART_UPDATE == 1 or SHOW_SHOPPING_CART_UPDATE == 3) ? zen_image_submit(ICON_IMAGE_UPDATE, ICON_UPDATE_ALT) : '') . zen_draw_hidden_field('products_id[]', $products[$i]['id']);
+  if (SHOW_SHOPPING_CART_UPDATE == 1 or SHOW_SHOPPING_CART_UPDATE == 3) {
+     if (!$showFixedQuantity) { 
+        $buttonUpdate = zen_image_submit(ICON_IMAGE_UPDATE, ICON_UPDATE_ALT); 
+     } else {
+        $buttonUpdate = zen_image_submit(ICON_IMAGE_UPDATE, ICON_UPDATE_ALT, 'style="opacity: 0.25" disabled="disabled"'); 
+     }
+  } else {
+     $buttonUpdate = ''; 
+  }
```

Comment:
I'd prefer that this final `else` clause be refactored (removed) by moving its operation up before the related `if`. 
ie: initialize it first (with the "else" bit). Then handle conditional logic to override it if appropriate. Therefore no need for `else` clause.